src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.gpiod_chip_close.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    private const string library = "libgpiod";
```

Comment:
Change the casing in this member, and also change the name as `library` is to general. Perhaps something like `LibGpiodLib` instead?